### DISCUSSION TITLE
Sensors values correction - temperature, humidity, etc. offset

### DISCUSSION THREAD

**Discussion by allmaechtig** [merdok/homebridge-miot#646] [Ideas]
This plugin is amazing .Thanks for the work so far. Can you please add support for sensor's correction? In example humidityCorrection, temperatureCorrection. Because Xiaomi devices do not support sensors calibration it would fix that problem.

**Comment by merdok** [comment]
I mean in theory it would be possible, i would just need to find some time to have a look at it...
